<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistorySettingsClass</class>
 <widget class="QWidget" name="HistorySettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HistorySettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QCheckBox" name="saveHistoryBox">
     <property name="text">
      <string>Save message history</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="recentBox">
     <property name="text">
      <string>Show recent messages:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="messagesCountBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>recentBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>messagesCountBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
